packages/ag-charts-enterprise/src/series/map-marker/mapMarkerSeries.ts

```diff
@@ -842,7 +843,61 @@ export class MapMarkerSeries
         );
     }
 
-    protected override computeFocusBounds(_opts: _ModuleSupport.PickFocusInputs): _Scene.BBox | undefined {
-        return undefined; // TODO
+    public getFormattedMarkerStyle(markerDatum: MapMarkerNodeDatum) {
```

Comment:
Note: a lot of the code in `getFormattedMarkerStyle` is taken from `updateMarkerNodes`. This should be tidied up, but for now I've just copy/pasted the code to avoid risky changes.